src/pages/testimonials.js

```diff
@@ -27,7 +27,7 @@ function Testimonial() {
             </div>
             <div className="col-md-12">
               <div className="cause_section_content">
-                <div className="testimonial-row">
+                <div className="testimonial-row" tabIndex="0">
```

Comment:
Please revert tab index on non interactive elements. 

https://www.a11yproject.com/posts/how-to-use-the-tabindex-attribute/

Reply:
Received, I will have it done tonight!

Reply:
This is resolved